.travis.yml

```diff
@@ -1,3 +1,5 @@
+before_install:
+  - gem update bundler
```

Comment:
If bundler is too old in travis, some issues might be occurred. To avoid trouble, update bundler before install.